tests/drivers/sensor/accel/src/main.c

```diff
@@ -14,8 +14,10 @@
 #include <zephyr/ztest.h>
 #include <zephyr/drivers/sensor.h>
 
-/* There is no obvious way to pass this to tests, so use a global */
-ZTEST_BMEM const struct device *dev;
+struct sensor_accel_fixture {
+	const struct device *accel0;
+	const struct device *accel1;
```

Comment:
I think it would make the test a little easier to read if you did this:
```suggestion
	const struct device *accel_spi;
	const struct device *accel_i2c;
```